### DISCUSSION TITLE
沃爾瑪擴大無人機送貨範圍

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#437] [General]
「沃爾瑪宣布，計劃增加提供無人機送貨的門市數量；未來將在六州、34 地空運貨物，包括亞利桑那州、阿肯色州、佛羅里達州、德州、猶他州和維吉尼亞州部分地區 400 萬戶家庭。這是顯著的增長，當 2021 年 11 月推出計畫時，還只有阿肯色州一城鎮提供服務。」

未來台灣也開放無人機的使用時希望像是一杯飲料之類的少量外送可以改成無人機運送，預想可以減少一部分因外送而導致的車流問題得到改善

https://technews.tw/2022/05/27/walmart-drone-deliveries-reach-4-million-homes/

**Comment by luo910227** [comment]
用無人機送貨實在挺創新的，但可能造成大量人員失業的問題也可能值得探討。
